algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableEvidenceFactory.java

```diff
@@ -100,43 +99,52 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
+                        if (drugClassInfo != null) {
+                            drugClass.add(drugClassInfo.drugClass());
                         }
-                    }
 
-                    String treatmentApproachString = String.join(",", sourceRelevantTreatmentApproaches);
-                    String treatmentApproachInterpret;
-                    if (sourceRelevantTreatmentApproaches.isEmpty()) {
-                        treatmentApproachInterpret = null;
-                    } else if (treatmentApproachString.endsWith(",")) {
-                        treatmentApproachInterpret = treatmentApproachString.substring(0, treatmentApproachString.length() - 1);
-                    } else {
-                        treatmentApproachInterpret = treatmentApproachString;
+                        if (therapyInfo != null) {
+                            therapy.add(therapyInfo.therapyName());
+                        }
                     }
-
-                    TreatmentApproachCurationEntryKey key = ImmutableTreatmentApproachCurationEntryKey.builder()
-                            .treatment(treatment)
-                            .treatmentApproach(treatmentApproachInterpret == null || treatmentApproachInterpret.isEmpty()
-                                    ? null
-                                    : treatmentApproachInterpret)
-                            .event(sourceGene + " " + entry.type())
-                            .direction(direction)
-                            .build();
-
-                    Set<String> curatedRelevantTreatmentApproaches = Sets.newHashSet(curator.isMatch(key));
+// TODO: implement
+//                    String treatmentApproachString = String.join(",", sourceRelevantTreatmentApproaches);
+//                    String treatmentApproachInterpret;
+//                    if (sourceRelevantTreatmentApproaches.isEmpty()) {
+//                        treatmentApproachInterpret = null;
+//                    } else if (treatmentApproachString.endsWith(",")) {
+//                        treatmentApproachInterpret = treatmentApproachString.substring(0, treatmentApproachString.length() - 1);
+//                    } else {
+//                        treatmentApproachInterpret = treatmentApproachString;
+//                    }
+//
+//                    TreatmentApproachCurationEntryKey key = ImmutableTreatmentApproachCurationEntryKey.builder()
+//                            .treatment(treatment)
+//                            .treatmentApproach(treatmentApproachInterpret == null || treatmentApproachInterpret.isEmpty()
+//                                    ? null
+//                                    : treatmentApproachInterpret)
+//                            .event(sourceGene + " " + entry.type())
+//                            .direction(direction)
+//                            .build();
+//
+//                    Set<String> curatedRelevantTreatmentApproaches = Sets.newHashSet(curator.isMatch(key));
```

Comment:
Is this still relevant? Maybe it can be deleted.

Reply:
I have annotated it with a TODO for implementation. This logica should be correctly implemented in another round so this cannot be deleted at this moment. However, this code will not be used at the moment.